### PR TITLE
TelephoneLink support for LinkButtonBlock

### PR DESCRIPTION
**Pull request by boehsermoe**

### CODE REVIEW

**Comment by nadar** [comment]
Looks good to me, but we can only (a.) merge or (.b) release this after LUYA core 1.0.9 has been released otherwise it will throw an exception. What would you prefer?

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ All notable changes to this project will be documented in this file. This projec
 + [#128](https://github.com/luyadev/luya-module-admin/issues/128) A new indicator display the amount of time left until the user is logged out automatically. Also every keystroke inside any text field will reset the logout timer to null. No more timeouts while working!
 + [#126](https://github.com/luyadev/luya-module-admin/issues/126) Provide option to eager load api model relations.
 + [#20](https://github.com/luyadev/luya-module-admin/issues/20) New option `--sync-requests-count` for proxy command.
++ [#144](https://github.com/luyadev/luya-module-admin/pull/144) Proxy command ask for large table sync.
```
Comment (nadar) [inline CHANGELOG.md]: @boehsermoe looks like you are mixing up between branches or sth. Cause this pr is about `TelephoneLink support for LinkButtonBlock`

**Comment by nadar** [comment]
@boehsermoe Seems you are missing branches. Maybe revert the last commit as it contains informations from sync max table prompt.

**Comment by boehsermoe** [comment]
Yes did it merge into wrong branch yesterday 😓

**Comment by nadar** [comment]
Well, as it looks like we need those features so i can trigger a release tomorrow if you like.

**Pull request merged**